client/src/utils/dateUtil.tsx

```diff
@@ -44,3 +50,89 @@ export const getHoursToDays = cacheWrapperForUnaryFunction((hours: number) => {
 
   return `${hours} hours`;
 });
+
+export const getTimeInSeconds = (endDate: string | null, startDate: string | null) => {
```

Comment:
getTimeDifferenceInSeconds = (startDate, endDate)

Reply:
Can we count the non working days and remove 48 hours for two days and 24 hours for one day to make this logic simple ?